src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -154,24 +152,42 @@ private function getIdentifiers(string $resourceClass, $id)
      * @param ObjectManager $resourceManager
      * @param string        $resourceClass
      *
+     * @throws RuntimeException
+     *
      * @return \Doctrine\ORM\QueryBuilder
      */
     private function retrieveQueryBuilder(ObjectManager $resourceManager, string $resourceClass)
     {
         $repository = $resourceManager->getRepository($resourceClass);
 
-        if ($repository instanceof  EntityRepository) {
+        if ($repository instanceof EntityRepository) {
             return $repository->createQueryBuilder('o');
         }
 
         if ($resourceManager instanceof EntityManagerInterface) {
```

Comment:
> $resourceManager could simply be a ObjectManager right?

Indeed, and the `ObjectManager` doesn't provide the `createQueryBuilder` method. `EntityManagerInterface` extends `ObjectManager`, and the `createQueryBuilder` method belongs to the `EntityManagerInterface`. 

As I'm no Doctrine expert, I've no further opinion on this. Hope others will :).